docs/rulesets.md

```diff
@@ -1,41 +1,128 @@
----
-title: Spectral Rulesets
-tags: 
-- Documentation
-- Rulesets
----
+# Spectral Rulesets
 
-Rulesets are a container for collections of rules. These rules are essentially calling functions Spectral, and by taking parameters you can make these functions do whatever you want. 
+Rulesets are a container for collections of rules. These rules are taking parameters, and calling functions on certain parts of the JSON object being linted.
 
-First the rule filters your object (JSON/YAML file) down to a set of target values, and then list what function arguments should be passed in.
+## CLI Usage
 
-## Usage
+When you run the `spectral lint my-document.json` CLI command, Spectral will automatically apply the built in OpenAPI v2 or v3 ruleset if appropriate.
 
-```bash
-spectral lint foo.yaml --ruleset=path/to/acme-company-ruleset.yaml --ruleset=http://example.com/acme-common-ruleset.yaml
+To customize the rules that are applied, simply create a `spectral.yml` in the same directory that you are running the `spectral lint` command from and it will automatically be used.
+
+## Ruleset Examples
+
+Spectral currently support ruleset files in both `yaml` and `json` formats.
+
+### Adding a rule
+
+Add your own rules under the `rules` property in your `spectral.yml` ruleset file.
+
+**spectral.yml**
+
+```yaml
+rules:
+  my-rule-name:
+    description: Tags must have a description.
+    given: $.tags[*]
+    severity: error
+    then:
+      field: description
+      function: truthy
+```
+
+The example above adds a single rule that checks that tags objects have a description property defined.
+
+Running `spectral lint` on the following object with the rulset above will result in an error being reported, since the tag does not have a description:
+
+```json
+{
+  "tags": [{
+    "name": "animals"
+  }]
+}
+```
+
+While running it with this object will succeed:
```

Comment:
I think this is missing a word or two between "object" and "will"

Reply:
Reads as intended to me - what do you suggest? Have to read it in the context of the whole section.

Reply:
"While running it with this object, it will succeed:"

Reply:
My understanding is that this is kind of a continuation of the previous sentence:

```
running spectral with this will fail…
…
…
while running it with this object will succeed
```

therefore, to me it looks correct. @tbarn ?

Reply:
It doesn't read as a sentence. You should see every new line as a new sentence in docs since people's eyes skip around and will miss context.

Reply:
All right!

Reply:
32ba8be